Cargo.toml

```diff
@@ -14,23 +14,16 @@ members = [
    "recover-id-object",
    "state-compare",
    "wallet-proxy-load-simulator",
-   "company-id-gui/src-tauri"
+   # Does currently not compile in CI, it can be added once it works
+   #"company-id-gui/src-tauri"
```

Comment:
I wonder how we ended up excluding it form being build in the first place which then probably lead to not update it anymore? What is the plan to fix it (specific issue)?